mods/cnc/maps/cnc64gdi01/en.ftl

```diff
@@ -0,0 +1,2 @@
+## rules.yaml
+actor-world-missiondata-briefing = Nod is experimenting on civilians with Tiberium. Use the commando to take out the SAM sites surrounding the dropoff area. With the SAMs gone you will then get an airstrike.  Take out the Obelisk and an MCV will be delivered to help you to locate and destroy the biochem facility.
```

Comment:
These key names are IMO very awkward. Since these are already scoped to the map, can we just call them `briefing`?